### DISCUSSION TITLE
need spam msg server

### DISCUSSION THREAD

**Discussion by geetee24** [afterxleep/Bouncer#67] [General]
how do i test the ability of this app?  I need a server out in www that is not in my contacts so that I can use it to send a spam msg to the bouncer app and see it filter it.  Know of one?

**Comment by glennzilla** [comment]
Did you/anyone figure this out?  I have the same need/desire.  I just installed this app and am setting up regex rules that I want to test but can't figure out how to generate text messages from unknown numbers.  Well, I did find various SMS spoofing services/APIs, usually requiring a license key or purchased passphrase, but I want to avoid using such untrusted and potentially nefarious services.

**Comment by afterxleep** [comment]
Although the app has internal test, during Development I use Twilio services for real-world manual testing.

I you just want to test regex matching, regex101.com is your best bet
